models/instrument.py

```diff
@@ -1569,7 +1579,7 @@ def preprocessing_calibrator_files( self, calibset, flattype, section, filter, m
                     # we had to do it before searching for the file so that we don't have a race
                     # condition for multiple processes all downloading the file at once.
                     calib = self._get_default_calibrator( mjd, section, calibtype=calibtype,
-                                                          filter=self.get_short_filter_name( filter ),
+                                                          filter=filter,
```

Comment:
Add in the docstring for this function that `filter` needs to be the short filter name, not the long filter name.